R/project_summary_response.R

```diff
@@ -0,0 +1,70 @@
+"dwapi-r
+Copyright 2017 data.world, Inc.
+
+Licensed under the Apache License, Version 2.0 (the \"License\");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an \"AS IS\" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+This product includes software developed at data.world, Inc.
+https://data.world"
+
+projectRequiredFields <- c(
+  "accessLevel",
```

Comment:
I'd like to be consistent with the use of quotes (single or double everywhere, but not mixed). Thanks.

Reply:
Done via commit 8bda938